packages/react-i18n/CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 <!-- ## [Unreleased] -->
 
+## [2.2.0] - 2019-??-??
+
+### Added
+
+- added `generateTranslationIndexes` helper and `mode=from-generated-index` option for babel plugin. This can be use to resolved caching issue when adding new translation file ([#1188](https://github.com/Shopify/quilt/pull/1188))
```

Comment:
```suggestion
- Added a `generateTranslationIndexes` helper, and `mode=from-generated-index` option for the Babel plugin. This can be use to resolve unwanted cache hits when adding new translation files ([#1188](https://github.com/Shopify/quilt/pull/1188))
```